### ISSUE TITLE
Bad month for LCD2004.

### ISSUE THREAD

**Issue by hevet** [Edzelf/ESP32-Radio#392]
The wrong date is showing on the LCD2004, it is one month earlier. If today is 17/10, it displays 17/09.
In the LCD2004.h file, change this solution in line 475 of the code:
timeinfo.tm_mon to timeinfo.tm_mon + 1.

**Comment by h1aji** [comment]
I found same issue here: https://github.com/Edzelf/ESP32-Radio/issues/382

**Comment by Edzelf** [comment]
Fixed.

**Comment by hevet** [comment]
> 
> 
> Fixed.

By mistake you subtracted another month instead of adding :)

**Comment by Edzelf** [comment]
Yes.  Will be fixed too....

**Comment by hevet** [comment]
There is one more problem. When we listen to the radio at midnight, the date and day of the week will not change. Only restarting the radio will refresh the day of the week and date. I don't know how it is on other displays.

**Comment by Edzelf** [comment]
Yes, the day is only updated once in 7200 seconds by connecting to an NTP server.
It was never my intention to display the date on a display (what's the point?), but this part was added by a user who wants to use the LCD2004 and display the date.  The preferred display is a 128x160 SPI display.

**Comment by hevet** [comment]
OK I understand.
